src/store/wallet/effects/send/send.ts

```diff
@@ -1351,6 +1351,29 @@ export const getTx = (wallet: Wallet, txpid: string): Promise<any> => {
   });
 };
 
+export const showNoCompatibleSelectedWalletModal =
+  (): Effect<void> => async dispatch => {
+    dispatch(
+      showBottomNotificationModal({
+        type: 'info',
+        title: t('No compatible wallet'),
+        message: t(
+          'Your selected wallet is not capable of sending this payment.',
```

Comment:
I would use something more clarifying for the user like: 

`Try again using a ${coin.toUpperCase()} wallet to pay this invoice`

Reply:
Nice idea. Thanks!